CodeReview/src/items/VReviewComment.cpp

```diff
@@ -39,22 +39,68 @@ DEFINE_ITEM_COMMON(VReviewComment, "item")
 VReviewComment::VReviewComment(Visualization::Item* parent, NodeType* node, const StyleType* style)
 	: Super{parent, node, style}
 {
+	date_ = new Visualization::Text{this,  ""};
+	username_ = new Visualization::Text{this, node->username()};
```

Comment:
So, unlike `date_` which needs some transformation from the thing stored in the node, to the thing that's displayed, `username_` is directly from the node. Instead of usint `Text`, use a `VText` and directly render the node.